app/src/main/java/no/usn/mob3000/ui/screens/chess/train/opening/OpeningsScreen.kt

```diff
@@ -78,15 +77,19 @@ fun OpeningsScreen(
     Viewport (
         floatingActionButton = {
             FloatingActionButton(
-                containerColor = DefaultButton,
+                containerColor = MaterialTheme.colorScheme.primary,
+                modifier = Modifier.shadow(15.dp),
```

Comment:
[See this comment](https://github.com/frigvid/mob3000-group11/pull/19/files#r1840963289).